apps/web/utils/gmail/filter.ts

```
@@ -21,19 +21,28 @@ export async function createFilter(options: {
-    removeLabelIds: [INBOX_LABEL_ID],
-    addLabelIds: gmailLabelId ? [gmailLabelId] : undefined,
-  });
+  try {
+    return await createFilter({
+      gmail,
+      from,
+      removeLabelIds: [INBOX_LABEL_ID],
+      addLabelIds: gmailLabelId ? [gmailLabelId] : undefined,
+    });
+  } catch (error) {
+    const errorMessage = (error as any)?.errors?.[0]?.message;
```

Biome

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)